__tests__/app.test.js

```diff
@@ -33,4 +33,42 @@ describe("/api/topics", () => {
                 expect(body.msg).toBe("path not found");
             });
     });
+});
+
+describe("/api/articles/:article_id", () => {
+    test("GET:200 should respond with the correct article object", () => {
+        const expectedArticle = {
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          }
+        return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toEqual(expectedArticle);
```

Comment:
I think you should refactor this test to be a little less 'brittle'.  It's likely we will add to the articles table later, so  the test should use a flexible matcher eg expect.objectContaining or .toMatchObject.  Testing for strict equality can lead to the test failing when it shouldn't further down the line, because of db changes.